backend/de.metas.business/src/main/sql/postgresql/system/20-de.metas.business/5681570_sys_gh14841_sap_gljournal_window_adj.sql

```diff
@@ -0,0 +1,103 @@
+;
+
+-- 2023-03-13T09:33:25.138Z
+UPDATE AD_Field_Trl trl SET Description=NULL,Name='Amount (LC)' WHERE AD_Field_ID=710039 AND AD_Language='en_US'
+;
+
+-- 2023-03-13T09:33:25.142Z
+/* DDL */  select update_FieldTranslation_From_AD_Name_Element(582136) 
+;
+
+-- 2023-03-13T09:33:25.148Z
+DELETE FROM AD_Element_Link WHERE AD_Field_ID=710039
+;
+
+-- 2023-03-13T09:33:25.150Z
+/* DDL */ select AD_Element_Link_Create_Missing_Field(710039)
+;
+
+-- 2023-03-13T09:34:58.735Z
+INSERT INTO AD_Element (AD_Client_ID,AD_Element_ID,AD_Org_ID,Created,CreatedBy,EntityType,IsActive,Name,PrintName,Updated,UpdatedBy) VALUES (0,582137,0,TO_TIMESTAMP('2023-03-13 10:34:58','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','GL Account','GL Account',TO_TIMESTAMP('2023-03-13 10:34:58','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2023-03-13T09:34:58.736Z
+INSERT INTO AD_Element_Trl (AD_Language,AD_Element_ID, CommitWarning,Description,Help,Name,PO_Description,PO_Help,PO_Name,PO_PrintName,PrintName,WEBUI_NameBrowse,WEBUI_NameNew,WEBUI_NameNewBreadcrumb, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Element_ID, t.CommitWarning,t.Description,t.Help,t.Name,t.PO_Description,t.PO_Help,t.PO_Name,t.PO_PrintName,t.PrintName,t.WEBUI_NameBrowse,t.WEBUI_NameNew,t.WEBUI_NameNewBreadcrumb, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Element t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Element_ID=582137 AND NOT EXISTS (SELECT 1 FROM AD_Element_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Element_ID=t.AD_Element_ID)
+;
+
+-- Field: GL Journal (SAP)(541656,de.metas.acct) -> GL Journal Line (SAP)(546731,de.metas.acct) -> GL Account
+-- Column: SAP_GLJournalLine.C_ValidCombination_ID
+-- 2023-03-13T09:35:04.097Z
+UPDATE AD_Field SET AD_Name_ID=582137, Description=NULL, Help=NULL, Name='GL Account',Updated=TO_TIMESTAMP('2023-03-13 10:35:04','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=710036
+;
+
+-- 2023-03-13T09:35:04.098Z
+UPDATE AD_Field_Trl trl SET Description=NULL,Help=NULL,Name='GL Account' WHERE AD_Field_ID=710036 AND AD_Language='en_US'
+;
+
+-- 2023-03-13T09:35:04.100Z
+/* DDL */  select update_FieldTranslation_From_AD_Name_Element(582137) 
+;
+
+-- 2023-03-13T09:35:04.103Z
+DELETE FROM AD_Element_Link WHERE AD_Field_ID=710036
+;
+
+-- 2023-03-13T09:35:04.103Z
+/* DDL */ select AD_Element_Link_Create_Missing_Field(710036)
+;
+
+-- Value: C_ValidCombination_ID
+-- 2023-03-13T09:58:05.877Z
+INSERT INTO AD_Message (AD_Client_ID,AD_Message_ID,AD_Org_ID,Created,CreatedBy,EntityType,IsActive,MsgText,MsgType,Updated,UpdatedBy,Value) VALUES (0,545256,0,TO_TIMESTAMP('2023-03-13 10:58:05','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','GL Account','I',TO_TIMESTAMP('2023-03-13 10:58:05','YYYY-MM-DD HH24:MI:SS'),100,'C_ValidCombination_ID')
```

Comment:
IMHO this is misleading, fragile and not intention revealing.

suggestion: introduce AD_Element `GL_Account_ID` (name: GL Account) and use it in SAPGLJournalLineQuickInputDescriptorFactory.createEntityDescriptor

Reply:
then pls remove this AD_Message.